met-web/src/components/engagement/admin/create/authoring/AuthoringSummary.tsx

```diff
@@ -83,62 +113,106 @@ const AuthoringBanner = () => {
+    const tryParse = (json: string) => {
+        try {
+            const object = JSON.parse(json);
+            if (object && typeof object === 'object') {
+                return object;
+            }
+        } catch (e) {}
+        return false;
     };
 
     return (
         <Grid container sx={{ maxWidth: '700px', mt: '1rem' }} direction="column">
             <Grid sx={{ ...formItemContainerStyles, backgroundColor: colors.surface.blue[10] }} item>
-                <label htmlFor="section_heading">
+                <label htmlFor="title">
                     <MetBigLabel style={metBigLabelStyles}>
                         Section Heading
                         <span style={{ fontWeight: 'normal' }}> (Required)</span>
                     </MetBigLabel>
                     <FormDescriptionText style={formDescriptionTextStyles}>
                         Your section heading should be descriptive, short and succinct.
                     </FormDescriptionText>
-                    <TextField
-                        sx={{ backgroundColor: colors.surface.white }}
-                        id="section_heading"
-                        counter
-                        maxLength={60}
-                        placeholder="Section heading text"
+                    <Controller
+                        control={control}
+                        name="title"
+                        rules={{ required: true }}
+                        render={({ field }) => {
+                            return (
+                                <TextField
+                                    {...field}
+                                    sx={{ backgroundColor: colors.surface.white }}
+                                    id="title"
+                                    value={field.value}
+                                    error={undefined}
+                                    counter
+                                    maxLength={60}
+                                    placeholder="Section heading text"
+                                    onChange={(value) => {
+                                        field.onChange(handleTitleChange(value));
+                                    }}
+                                />
+                            );
+                        }}
                     />
                 </label>
             </Grid>
 
             <Grid sx={{ ...formItemContainerStyles, backgroundColor: colors.surface.blue[10] }} item>
-                <label htmlFor="body_copy">
+                <label htmlFor="editor_state">
                     <MetBigLabel style={metBigLabelStyles} role="document" tab-index="0">
                         Body Copy
                         <span style={{ fontWeight: 'normal' }}> (Required)</span>
                     </MetBigLabel>
                     <FormDescriptionText style={formDescriptionTextStyles}>
                         Your section heading should be descriptive, short and succinct.
```

Comment:
Funny story, the state of the RichTextArea was very hard to manage because of the Controller. However, I need the Controller because it checks if the form is dirty. So this is actually a dummy field that isn't sent to the API, but it manages the state of the RichTextArea. However, I think I should rename it to something like summary_editor_state because that would be specific to the summary page.

Reply:
I'm curious about this and I'll message you to learn more!

Reply:
Sorry I thought we were talking about the editor_state label but you meant the FormDescriptionText text. Fixed.